aip/0203.md

```diff
@@ -73,7 +73,8 @@ error. A corollary to this is that a required boolean must be set to `true`.
 The use of `OUTPUT_ONLY` indicates that the field is provided in responses, but
 that including the field in a message in a request does nothing (the server
 **must** ignore it and **must not** throw an error as a result of the presence
-of a value in this field on input).
+of a value in this field on input). Similarly, services **must** ignore the
+presence of output only fields in field masks.
```

Comment:
Doesn't this depend on the context of the field mask? Some may be "Please only give me these fields in the response" rather than being used for update, for example - in which case a service should absolutely take note of which output only fields are used!

Reply:
Fair point.

```suggestion
presence of output only fields in update field masks.
```